westcoast/src/main/java/frc/commands/drive/LeaveLine.java

```diff
@@ -19,12 +19,12 @@
     private int wait = 0;
 
     public LeaveLine(
-            double encoderTicks,
+            double distance,
             @NotNull ProfiledPIDController left,
             @NotNull ProfiledPIDController right,
             @NotNull DriveTrainSubsystem drive
     ) {
-        this.encoderTicks = encoderTicks;
+        this.encoderTicks = distance * 64.0;
```

Comment:
if we keep using ticks here why not just accept in encoder tick units whatever a tick is?
I would say if you really think that is a must have necessity (I don't see as such for a few reasons) we should keep accepting ticks in the constructor and have a function somewhere else that make the conversion.

Reply:
I am going to fix this natively over the weekend (get distance will return in meters [wpilib's preferred unit]) so we can then tune stuff using their tuners.